src/store/socket_init.js

```diff
@@ -1,31 +1,77 @@
+/* eslint no-param-reassign: ["error", { "props": false }] */
 import io from 'socket.io-client';
-import * as actions from './Chat/actions.js';
-// import actions that have anything to do with sockets.
+import {
```

Comment:
any reason you prefer writing out the entire import list rather than importing as `*`? consistency across the code base? 

I think I did it as it seemed like a lot to write out. Just curious.